data/customerCounts.json

```diff
@@ -0,0 +1,301 @@
+[{
+	"companyId": "9e0be953-bd9e-4949-9991-c16756ab548f",
+	"companyName": "Yakijo",
+	"lastPurchase": "2015-10-05",
+	"isReminded": false,
+	"purchaseCount": 7
+}, {
```

Comment:
Slightly different data structure. ```purchaseCount``` is the (fake) number of invoices received from this company (so it doesn't have to be calculated in the frontend). @schuurthing